### ISSUE TITLE
Settings: Sub-chain - add a sub-chain 1/4

### ISSUE THREAD

**Issue by crsfalzon** [SimplyVC/panic#224]
### User Story
As a node operator, I want to be able to add a new sub-chain, so that I can easily manage and grow my setup.

### Description
The scope of this task is to implement the functionality that will allow a node operator to monitor and be alerted on a new sub-chain.

![update chain a](https://user-images.githubusercontent.com/89395527/171128938-243895bf-b4ce-4f10-a01e-ca1cffe6e4c6.PNG)

### Requirements
- The node operator lands on this page after clicking on Chains from the Settings dropdown #223
- Title reading Sub-chains
- 'Add a sub-chain' button
- Clicking on the button takes the node operator to the **Blockchain setup step of the installer** (step 1 of 5) (_/installer/sub-chains-setup_). From here the node operator can start the installer journey that ends with the Success page #188 
- Include footer section.

### Blocked by
Not dependent on another ticket to be tackled.

### Acceptance criteria
**Scenario**: The node operator is on _configuration/sub-chains_ after having selected Chains from the Settings dropdown #223 
**Given**: The node operator wants to add a sub-chain for monitoring 
**When**: He clicks on the Add a sub-chain button
**Then**: He starts the installer journey from **step 1/5** (_/installer/sub-chains-setup_).
